src/main/java/org/codehaus/mojo/jaxws/AbstractJaxwsMojo.java

```diff
@@ -439,9 +439,16 @@ protected void exec( List<String> arguments )
                 }
             }
             InvokerCP classpath = getInvokerCP();
-            cmd.createArg().setValue( "-Xbootclasspath/p:" + classpath.ecp );
-            cmd.createArg().setValue( "-cp" );
-            cmd.createArg().setValue( classpath.invokerPath );
+            if (isPre9()) {
+                cmd.createArg().setValue( "-Xbootclasspath/p:" + classpath.ecp );
+                cmd.createArg().setValue( "-cp" );
+                cmd.createArg().setValue( classpath.invokerPath );
+            } else {
+                cmd.createArg().setValue( "-cp" );
+                cmd.createArg().setValue( classpath.ecp + File.pathSeparator + classpath.invokerPath );
+                cmd.createArg().setValue( "--add-modules");
+                cmd.createArg().setValue( "java.activation");
```

Comment:
the module `java.activation` is marked as deprecated and will be removed in the Java 11. Instead of adding this module it's better to add dependency `com.sun.activation : javax.activation` (https://search.maven.org/#artifactdetails%7Ccom.sun.activation%7Cjavax.activation%7C1.2.0%7Cjar)